### PR TITLE
Add JC65 ps2avrGB keyboard

### PR DESCRIPTION
**Pull request by nautxx**
Left the backlight support out for now. I tried and couldn't get the rgb animations to work properly with the backlighting code in. The static rgb was still functional with it in. I named it jc65p since there is already a qmk version of the jc65, but if there is something better to call, it let me know.

### CODE REVIEW

**Comment by jackhumbert** [comment]
Cool! I think'd it make sense to group these together - is that something you'd be willing to do?

**Comment by nautxx** [comment]
I should have clarified that the 2 versions of JC65's use 2 different MCU's. The qmk version uses the atmega32u4 and this version uses the atmega32a. Should I still be grouping them together? This is actually my first time doing all this.

**Comment by jackhumbert** [comment]
Yeah! You can define that in the respective `rules.mk` files for each.

**Comment by nautxx** [comment]
Well then. I will work on that ;)

**Comment by mechmerlin** [comment]
It would be good to have it as a jc65/32u4 directory for the OG QMK stuff already in there, and a jc65/32a for  your stuff, and then just a brief description of that in the readme.md file.

**Comment by nautxx** [comment]
Coming up. Thanks gentlemen!

**Comment by mechmerlin** [comment]
you make want to check how I did it in the octagon/v2 and octagon/v1 directories. This way an end user can just type in octagon/v1:default or octagon/v2:default. In the same way for your files it should be make jc65/32u4:default or make jc65/32a:default.

**Comment by nautxx** [comment]
Thank you for that! I will try that out tomorrow!

**Comment by nautxx** [comment]
Alright. How does this look?

File: keyboards/jc65/v32a/config.h [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+*/
+
+#ifndef CONFIG_H
+#define CONFIG_H
+
+#include "config_common.h"
+
+#define VENDOR_ID       0x1234
+#define PRODUCT_ID      0x5679
+#define MANUFACTURER    winkeyless.kr
```
Comment (mechmerlin) [inline keyboards/jc65/v32a/config.h]: I was unaware that winkeyless actually made the PCB. News to me!

**Comment by mechmerlin** [comment]
Overall, I lilke it!

**Comment by jackhumbert** [comment]
Awesome! Looks good :) Thanks!

**Pull request merged**